### PR TITLE
Fix ConfirmationDialog on ListView showing raw HTML

### LINKED ISSUE #3376: Modal renders raw HTML
With my fix for some prop type warnings in #3358 I introduced a regression I didn't see, the `ModalBody` now renders some HTML strings in certain cases:

screen shot 2016-08-23 at 16 22 00

### PR DESCRIPTION
**Pull request by mxstbr**
Closes #3376

### CODE REVIEW

File: admin/client/App/screens/List/index.js [hunk 1]
```diff
@@ -339,7 +339,7 @@ const ListView = React.createClass({
 			confirmationDialog: {
 				isOpen: true,
 				label: 'Delete',
-				body: `Are you sure you want to delete <strong>${item.name}</strong>?<br /><br />This cannot be undone.`,
+				body: <p>Are you sure you want to delete <strong>${item.name}</strong>?<br /><br />This cannot be undone.</p>,
```
Comment (JedWatson) [inline admin/client/App/screens/List/index.js]: That's pretty close to being multiline JSX, should we break it out and wrap it in `()` for readability?

Comment (mxstbr) [inline admin/client/App/screens/List/index.js]: Sure

Comment (mxstbr) [inline admin/client/App/screens/List/index.js]: Done

**Comment by JedWatson** [comment]
:+1:

**Pull request merged**